vision/realsense/read_bag.py

```diff
@@ -0,0 +1,113 @@
+"""
+Test program from the examples in the RealSense repo (at
+https://github.com/IntelRealSense/librealsense/tree/master/wrappers/python/examples)
+to read .bag files into a python program
+
+For the program to work, you must enter the correct resolution width, height,
+framerate, depth stream type, and color stream type
+"""
+
+
+# First import library
+import pyrealsense2 as rs
+# Import Numpy for easy array manipulation
+import numpy as np
+# Import OpenCV for easy image rendering
+import cv2
+# Import argparse for command-line options
+import argparse
+# Import os.path for file path manipulation
+import os.path
+
+def read_from_bag(width, height, framerate, depth_type, color_type):
```

Comment:
Have default values for each of these
ie width=1280, height=720, framerate=30, depth_type='z16', color_type='rgb8'